mint-001.md

```diff
@@ -0,0 +1,248 @@
+# mint-001
+
+## 3 Key Degrading Multisig
+
+## Motivation
+
+The 2-of-3 multisig serves as a standard in multisignature solutions, offering a well-balanced combination of redundancy, security, flexibility, and practicality for securing Bitcoin. Loss of a single key does not spell disaster, and key distribution enhances security. Miniscript's time feature enables further disaster recovery. If two keys are lost, a timelock can convert the wallet to a 1-of-3 multisig. The proposal outlines four conditions through Miniscript's `thresh()`:
+
+### 2 Layer Conditions to be satisfied using:
+
+#### Base expressions (type B).
+
+##### timelock: **relative** or **absolute** [^timelock] [^either]
+
+- **older(**int**)** [^older]
+
+- **after(**int**)** [^after]
+
+#### Key expressions[^k_type] (type K).
+
+- **pk(**$Key_1$**)**
+
+- **pk(**$Key_2$**)**
+
+- **pk(**$Key_3$**)**
+
+Initially, the wallet requires 2-of-3 keys, functioning as a traditional 2-of-3 multisig. After the timelock expires, only one key is needed to spend the funds.
+
+### More on Timelock Values
+
+- There are no valid timelock values that change the security or correctness properties beyond those implied by the [relative](mint-000.md#absolute-block-height-based-timelocks) or [absolute](mint-000.md#block-height-based-timelocks) timelocks themselves. The reference transactions on testnet [^reftx1] [^reftx2] [^reftx3] [^reftx4] use short-duration timelocks. However, practical applications will vary.
+
+- Relative timelock descriptors provide a persistent structure, requiring a self-send to extend timelock security, thus enabling standardized timelock durations.
+
+- Absolute timelock descriptors need frequent updates; setting long-term timelock values becomes cumbersome. Updating these descriptors entails transferring coins to a new descriptor with revised absolute timelock values.
+
+##### Suggested Relative Block Height Timelocks:
+
+- **older(**32800**)** - Halfway point of block height relative timelock [^278days]
+
+- **older(**65535**)** - Maximum duration of a block height relative timelock [^455days]
+
+##### Suggested Relative Epoch Timelocks:
+
+- **older(**4224680**)** - Approximate Halfway point of epoch time relative timelock[^180days]
+
+- **older(**4259839**)** -- Maximum duration of an epoch time relative timelock[^388days]
+
+##### Suggested Relative Epoch Timelocks: older(4224679)
+
+- Mid-point epoch time relative timelock (\~180 days, 6 months) older(4259839)
+
+- Max duration epoch time relative timelock (\~388 days)
+
+	- It is suggested that the first **Relative Epoch Timelock** occurs prior to the next. "Cascading" the time locks sequentially is not necessary but eases reasoning about the intended purpose and resulting [Bitcoin Script](https://en.bitcoin.it/wiki/Script) structure.
+
+<!--
```

Comment:
I introduce the terminology "Cascading TimeLock" and give a short explanation on why TimeLocks should be sequential.